TACC/Core-Portal
# Bug/wp 47 fix shared workspace manage team modal

## Overview
In 'Shared Workspace':
1. if user is a GUEST, disable 'Rename' and 'Trash' buttons
2. change button style for 'Update' button to match toolbar

## Related

* [WP-47](https://tacc-main.atlassian.net/browse/WP-47)

## Changes
1. In **DataFilesToolbar.jsx**: created an '`isGuest`' field which uses the `authenticatedUserQuery` to determine if a user is a guest. `canRename` and `canTrash` check `isGuest`; if that is True, then those buttons are disabled. 
2. In **SystemRoleSelector.jsx**: changed the button import to come from _common. Added a left margin to the css (DataFilesProjectMembers.module.scss)

## Testing
1. Login as a guest, go to a Data Files > Shared Workspace, verify that the 'Rename' and 'Trash' buttons are disabled.
2. Login normally, edit the Team Members for a Shared Workspace, verify the button is matches the buttons in the toolbar and turns purple when rolled over. 

## UI
Disable 'Rename' and 'Trash' for Guest
https://github.com/TACC/Core-Portal/assets/35277477/b41396be-86bc-4d2a-80ae-fadeaff350c5

changes to 'Update' button
<img width="1248" alt="Screen Shot 2023-12-18 at 3 49 09 PM 1" src="https://github.com/TACC/Core-Portal/assets/35277477/e85654bc-5616-4cde-a86a-61dfd0e2cf11">

## Notes
